### PR TITLE
Decentraland (Independent Publisher)

### PR DESCRIPTION
**Pull request by RoyPaarMS**
Initial Commit

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
connector 2

- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
connector 1

- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/Decentraland/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Publisher: Roy Paar
+
+## Prerequisites
+None.
+
+## Supported Operations
+### Get Districts
+Get a list of all districts within Decentraland.  Returns all parcels within each district with x, y coordinates.
+
```
Comment (v-eisljami) [inline independent-publisher-connectors/Decentraland/readme.md:12]: There is no Obtaining Credentials section. Please refer to this [link ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)to see the official template which you should follow.

Comment (RoyPaarMS) [reply independent-publisher-connectors/Decentraland/readme.md:15]: Fixed

**Review by v-eisljami — APPROVED** [review]

**Comment by v-eisljami** [comment]
[[certify-connector]]

**Review by raghavthind2005 — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hi @RoyPaarMS, I hope you are doing well.
Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fforms.office.com%2FPages%2FResponsePage.aspx%3Fid%3Dv4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Ca8689fe38e534722c8bb08da1e887ceb%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637855868806222105%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=XLaJOJaAoUppfgAo%2FXfFV6hTsrxE6uZX%2B9TRpdq0esQ%3D&reserved=0) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much.

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**